Pod/Classes/Cards/Card.swift

```diff
@@ -13,15 +13,32 @@ import Foundation
 */
 public struct Card {
 
+    /// The card number that is shown on the user's card.
     public let bankCardNumber: Number
+    
+    /**
+     The card verification code for the user's card.
+     
+     - important: You should never save and/or autofill a user's card verification code in your app, as this will void the CVC's purpose as a security feature.
+     - important: CVCs that are validated by *Caishen* as `CardValidationResult.Valid` are not necessarily validated by the card issuer as well. In order to validate the CVC, you should opt for the integration of a [Payment Gateway](https://en.wikipedia.org/wiki/Payment_gateway) to validate the authenticity of the entered CVC.
```

Comment:
not sure about this but if you provide external links it should be in the `seealso` section: https://developer.apple.com/library/mac/documentation/Xcode/Reference/xcode_markup_formatting_ref/SeeAlso.html#//apple_ref/doc/uid/TP40016497-CH45-SW1

> Use the callout to add references to other information.